recipes-wpe/wpeframework/wpeframework-plugins-rdk_git.bb

```diff
@@ -9,5 +9,7 @@ PV = "3.0+git${SRCPV}"
 RECIPE_BRANCH ?= "master"
 SRC_URI = "git://git@github.com:/WebPlatformForEmbedded/ThunderNanoServicesRDK.git;protocol=ssh;branch=${RECIPE_BRANCH}"
 SRCREV ?= "65eb1c658391580447c7c93b890cfee772e058bf"
-
 require include/wpeframework-plugins-rdk.inc
+
+ASNEEDED = ""
```

Comment:
Why it is needed now?

Reply:
It is required for WPEWebkit 2.28 external library case.